apisix/plugins/real-ip.lua

```diff
@@ -84,6 +104,22 @@ local function get_addr(conf, ctx)
             return addrs
         end
 
+        if conf.recursive and conf.trusted_addresses then
+            local split_addrs = {}
+            local match_itr = str_gmatch(addrs, "[^,%s*]+")
```

Comment:
can we use `ngx.re.gmatch` to replace `string.gmatch`

Reply:
Solved.